universal-login-react/src/core/models/ReactDashboardContentType.ts

```diff
@@ -1 +1 @@
-export type DashboardContentType = 'funds' | 'topup' | 'transferAmount' | 'transferRecipient' | 'waitingForTransfer' | 'devices' | 'backup' | 'none';
+export type DashboardContentType = 'funds' | 'topup' | 'transferAmount' | 'waitingForTransfer' | 'devices' | 'backup' | 'none';
```

Comment:
Should we rename `transferAmount` to simply `transfer`?

Reply:
It turns out `DashboardContentType` is not used anywhere. Deleted.